src/picknik_ur_gazebo_config/objectives/teach_apriltag_grasp_offset.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<root BTCPP_format="4" main_tree_to_execute="Calculate Offset From Apriltag">
+    <BehaviorTree ID="Calculate Offset From Apriltag" _description="Saves a grasp offset to yaml file from Apriltag observation using an interactive marker">
```

Comment:
This objective should be named "Teach Grasp Offset From AprilTag" but we will need to make a PR to Studio so that the gripper interactive marker continues to work. Right now it's hard coded to look for this objective name.